jest.config.ts

```diff
@@ -1,13 +1,9 @@
 import type { Config } from "jest";
 
 const cfg: Config = {
+  testEnvironment: "node",
   transform: {
-    "^.+\\.tsx?$": [
-      "ts-jest",
-      {
-        useESM: true,
-      },
-    ],
+    "^.+\\.[jt]s$": ["@swc/jest", {}],
```

Comment:
```suggestion
    "^.+\\.[jt]s$": "@swc/jest",
```

should also work fine right?